src/config.rs

```diff
@@ -341,7 +341,7 @@ pub enum PolicyOrPolicyGroup {
     #[serde(rename_all = "camelCase")]
     Policy {
         /// The URL where the policy is located
-        url: String,
+        module: String,
```

Comment:
what about using `#[serde(alias = "url")]` so we do not break the current behavior?

Reply:
Thinking it over, this only affects raw policy users, so I'm fine with the breaking changes.

Reply:
Opened https://github.com/kubewarden/docs/pull/484 for updating the user-facing docs.